std/internal/math/gammafunction.d

```diff
@@ -795,7 +796,8 @@ real beta(in real x, in real y)
     assert(beta(nextUp(-0.5L), 0.5) < 0);
     assert(beta(-0.5, nextUp(0.5L)) < 0);
     assert(beta(-0.5, real.infinity) == -real.infinity);
-    assert(cmp(beta(nextDown(-0.0L), 2*nextUp(+0.0L)), -0.0L) <= 0);
+    static if (real.mant_dig != 113) // TODO: yields 1 with quadruple precision, verify!
+        assert(cmp(beta(nextDown(-0.0L), 2*nextUp(+0.0L)), -0.0L) <= 0);
```

Comment:
Unsure about this one. **Edit**: The `beta(…)` result is a NaN apparently.

Reply:
Changed to `version(AArch64)`, as `beta()` yields a positive NaN for macOS arm64 CI too, with 64-bit `real`. Note that this AArch64/ARM(?) behavior isn't affected by setting the non-fast FPU mode in https://github.com/dlang/dmd/blob/c8e2ae291414f839de5a7635da09a03c63fed2f6/druntime/src/test_runner.d#L114-L152.